[2022-04-18 15:28:34,393] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.processing_tasks.training_model_a 2022-04-17T00:00:00+00:00 [queued]>
[2022-04-18 15:28:34,742] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: xcom_dag.processing_tasks.training_model_a 2022-04-17T00:00:00+00:00 [queued]>
[2022-04-18 15:28:34,743] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-04-18 15:28:34,743] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2022-04-18 15:28:34,744] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-04-18 15:28:35,053] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): processing_tasks.training_model_a> on 2022-04-17T00:00:00+00:00
[2022-04-18 15:28:35,113] {standard_task_runner.py:52} INFO - Started process 3596 to run task
[2022-04-18 15:28:35,153] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'xcom_dag', 'processing_tasks.training_model_a', '2022-04-17T00:00:00+00:00', '--job-id', '129', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/xcom_dag.py', '--cfg-path', '/tmp/tmpotsb_jle', '--error-file', '/tmp/tmp0n_hi1sx']
[2022-04-18 15:28:35,255] {standard_task_runner.py:77} INFO - Job 129: Subtask processing_tasks.training_model_a
[2022-04-18 15:28:36,243] {logging_mixin.py:104} INFO - Running <TaskInstance: xcom_dag.processing_tasks.training_model_a 2022-04-17T00:00:00+00:00 [running]> on host airflowvm
[2022-04-18 15:28:36,998] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=xcom_dag
AIRFLOW_CTX_TASK_ID=processing_tasks.training_model_a
AIRFLOW_CTX_EXECUTION_DATE=2022-04-17T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-17T00:00:00+00:00
[2022-04-18 15:28:37,012] {logging_mixin.py:104} INFO - model's accuracy: 3.6769494502985722
[2022-04-18 15:28:37,612] {python.py:151} INFO - Done. Returned value was: None
[2022-04-18 15:28:37,889] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=xcom_dag, task_id=processing_tasks.training_model_a, execution_date=20220417T000000, start_date=20220418T152834, end_date=20220418T152837
[2022-04-18 15:28:38,502] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-04-18 15:28:38,549] {local_task_job.py:151} INFO - Task exited with return code 0
